### PR TITLE
Add cones

### PR DESCRIPTION
**Pull request by TiagoCavalcante**

### CODE REVIEW

File: src/primitives/cone.js [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+	// The lines below are the edges' vertices' positions.
+	// The magic numbers below are modified from the position attribute of a three.js EdgesGeometry of the cone.
+	// To get them: console.log(new EdgesGeometry(coneGeometry).attributes.position.array)
+
+	const edgesGeometry = new InstancedBufferGeometry()
+		.setAttribute(
+			'position',
+			new BufferAttribute(
+				new Float32Array([
+					0, -1, 0,
```
Comment (rocky) [inline src/primitives/cone.js:129]: What's a breakout of the structure of these coordinates?

**Review by rocky — APPROVED** [review]
LGTM Tried it out the example and it looks great!

File: src/primitives/cone.js (hunk 1 above)
Comment (TiagoCavalcante) [reply src/primitives/cone.js:130]: This is the cone base, I changed the variables' names to make this more clear, and changed a comment.

**Comment by TiagoCavalcante** [comment]
I improved this so it uses less RAM and is faster.

**Comment by TiagoCavalcante** [comment]
A good part of this code is almost the same than the code from cylinders, but I don't think should be moved to another file.

I'm mergingin it now. Refactors may come in new PRs.

**Pull request merged**